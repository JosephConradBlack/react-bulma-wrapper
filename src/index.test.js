import initDoczPluginSnapshots from 'docz-plugin-snapshots';
initDoczPluginSnapshots('./');
