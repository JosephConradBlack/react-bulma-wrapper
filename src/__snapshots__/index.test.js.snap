// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container_0 1`] = `
<div
  className="container"
>
  <div
    style={
      Object {
        "backgroundColor": "lightblue",
      }
    }
  >
    This container is centered on desktop
  </div>
</div>
`;

exports[`Container_1 1`] = `
<div
  className="container is-fluid"
>
  <div
    style={
      Object {
        "backgroundColor": "lightblue",
      }
    }
  >
    This container is 
    <strong>
      fluid
    </strong>
    : it will have a 32px gap on either side, on any viewport size.
  </div>
</div>
`;

exports[`Container_2 1`] = `
<div
  className="container is-fluid"
>
  <div
    style={
      Object {
        "backgroundColor": "lightblue",
      }
    }
  >
    This container is fullwidth until the widescreen breakpoint.
  </div>
</div>
`;

exports[`Container_3 1`] = `
<div
  className="container is-fullhd"
>
  <div
    style={
      Object {
        "backgroundColor": "lightblue",
      }
    }
  >
    This container is fullwidth until the FullHd breakpoint.
  </div>
</div>
`;

exports[`Hero_0 1`] = `
<section
  className="hero is-primary"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Title
      </h1>
      <h2
        className="subtitle"
      >
        Subtitle
      </h2>
    </div>
  </div>
</section>
`;

exports[`Hero_1 1`] = `
<section
  className="hero is-info"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Title
      </h1>
      <h2
        className="subtitle"
      >
        Subtitle
      </h2>
    </div>
  </div>
</section>
`;

exports[`Hero_2 1`] = `
<section
  className="hero is-primary is-fullheight"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Title
      </h1>
      <h2
        className="subtitle"
      >
        Subtitle
      </h2>
    </div>
  </div>
</section>
`;

exports[`Navbar_0 1`] = `
<ul>
  <li>
    <strong>
      Navbar
    </strong>
      - The main navbar container
  </li>
  <li>
    <strong>
      Navbar.Brand
    </strong>
     - The left-side of the navbar. This component and it's children are always visible. This can optionally display a 'Hamburger' menu icon.
  </li>
  <li>
    <strong>
      Navbar.Menu
    </strong>
     - The right-side of the navbar. Hidden on small devices, visible on desktop
  </li>
  <li>
    <strong>
      Navbar.Start
    </strong>
     - The left part of the Navbar.Menu. This will appear next to the navbar brand on desktop
  </li>
  <li>
    <strong>
      Navbar.End
    </strong>
     - The right part of the Navbar.Menu. This will appear at the end of the navbar.
  </li>
  <li>
    <strong>
      Navbar.Item
    </strong>
     - Each individual item present on the Navbar.
  </li>
</ul>
`;

exports[`Navbar_1 1`] = `
<nav
  aria-label="main navigation"
  className="navbar is-primary"
>
  <div
    className="navbar-brand"
  >
    <a
      className="navbar-item"
    >
      Brand
    </a>
    <a
      aria-expanded="false"
      aria-label="menu"
      className="navbar-burger"
      role="button"
    >
      <span
        aria-hidden="true"
      />
      <span
        aria-hidden="true"
      />
      <span
        aria-hidden="true"
      />
    </a>
  </div>
  <div
    className="navbar-menu"
  >
    <div
      className="navbar-start"
    >
      <a
        className="navbar-item"
      >
        Start 1
      </a>
      <a
        className="navbar-item"
      >
        Start 2
      </a>
    </div>
    <div
      className="navbar-end"
    >
      <a
        className="navbar-item"
      >
        End 1
      </a>
      <a
        className="navbar-item"
      >
        End 2
      </a>
    </div>
  </div>
</nav>
`;

exports[`Title_0 1`] = `
<ul>
  <li>
    Title
  </li>
  <li>
    Subtitle
  </li>
</ul>
`;

exports[`Title_1 1`] = `
<h1
  className="title"
>
  Title
</h1>
`;

exports[`Title_2 1`] = `
<h2
  className="subtitle"
>
  Subtitle
</h2>
`;

exports[`Title_3 1`] = `
<h1
  className="title is-size-1"
>
  Title
</h1>
`;

exports[`Title_4 1`] = `
<h1
  className="title is-size-2"
>
  Title
</h1>
`;

exports[`Title_5 1`] = `
<h1
  className="title is-size-3"
>
  Title
</h1>
`;

exports[`Title_6 1`] = `
<h1
  className="title is-size-4"
>
  Title
</h1>
`;

exports[`Title_7 1`] = `
<h1
  className="title is-size-5"
>
  Title
</h1>
`;

exports[`Title_8 1`] = `
<h1
  className="title is-size-6"
>
  Title
</h1>
`;

exports[`Title_9 1`] = `
<h1
  className="title is-size-7"
>
  Title
</h1>
`;

exports[`Title_10 1`] = `
<h2
  className="subtitle is-size-1"
>
  Subtitle
</h2>
`;

exports[`Title_11 1`] = `
<h2
  className="subtitle is-size-2"
>
  Subtitle
</h2>
`;

exports[`Title_12 1`] = `
<h2
  className="subtitle is-size-3"
>
  Subtitle
</h2>
`;

exports[`Title_13 1`] = `
<h2
  className="subtitle is-size-4"
>
  Subtitle
</h2>
`;

exports[`Title_14 1`] = `
<h2
  className="subtitle is-size-5"
>
  Subtitle
</h2>
`;

exports[`Title_15 1`] = `
<h2
  className="subtitle is-size-6"
>
  Subtitle
</h2>
`;

exports[`Title_16 1`] = `
<h2
  className="subtitle is-size-7"
>
  Subtitle
</h2>
`;

exports[`Typography_0 1`] = `
<table>
  <thead>
    <tr>
      <th
        align="left"
      >
        Property
      </th>
      <th
        align="center"
      >
        Type
      </th>
      <th
        align="right"
      >
        Description
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td
        align="left"
      >
        centered
      </td>
      <td
        align="center"
      >
        bool
      </td>
      <td
        align="right"
      >
        Center all children text content
      </td>
    </tr>
    <tr>
      <td
        align="left"
      >
        justified
      </td>
      <td
        align="center"
      >
        bool
      </td>
      <td
        align="right"
      >
        Justify all children text content
      </td>
    </tr>
    <tr>
      <td
        align="left"
      >
        left
      </td>
      <td
        align="center"
      >
        bool
      </td>
      <td
        align="right"
      >
        Align all children text content to the left
      </td>
    </tr>
    <tr>
      <td
        align="left"
      >
        right
      </td>
      <td
        align="center"
      >
        bool
      </td>
      <td
        align="right"
      >
        Align all children text content to the right
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`Typography_1 1`] = `
<div
  className="container has-text-centered"
>
  <div
    style={
      Object {
        "backgroundColor": "lightblue",
      }
    }
  >
    Center aligned text
  </div>
</div>
`;

exports[`index_0 1`] = `
<section
  className="hero is-primary"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Title
      </h1>
      <h2
        className="subtitle"
      >
        Subtitle
      </h2>
    </div>
  </div>
</section>
`;
